lib/features/children/add_member/widgets/add_member_loading_page.dart

```diff
@@ -29,14 +29,16 @@ class AddMemberLoadingPage extends StatelessWidget {
             SnackBarHelper.showMessage(context, text: state.error);
           } else if (state.status == AddMemberStateStatus.successCached) {
             _navigateToProfileSelection(context);
-          } else if (state.status == AddMemberStateStatus.successNoAllowances) {
```

Comment:
We don't need an if case for the successNoAllowance anymore?

Reply:
i deleted both AddMemberStateStatus.successNoAllowances and AddMemberStateStatus.successNoTopups